src/en/components/radio/base.md

```diff
@@ -5,8 +5,10 @@ figma: https://www.figma.com/file/mh2maMG2NBtk41k1O1UGHV/Canadian-Digital-Servic
 permalink: false
 tags: ['radioEN', 'header']
 ---
-
 # Radio <br>`<gcds-radio>`
+<gcds-error-message message="This component is deprecated as of v0.21.0"></gcds-error-message>
+
+Please see <gcds-link href="https://github.com/cds-snc/gcds-components/releases/tag/gcds-components-v0.21.0">Release v0.21.0</gcds-link> for details
```

Comment:
Should we move this sentence into the error message to keep all the relevant information about the component being deprecated in one place and highlighted?

Reply:
I tried that but I couldn't use a link because the message is an attribute and not a slot at the moment 🥺

Reply:
We'll change it after that error-message change is live! :)